### PR TITLE
Training module

### PR DESCRIPTION
**Pull request by taracampbell**
Separating the training module from the examiner module.

- Adds a basic integration test for the training module

Caveat: removes the H4 styling from the main.css because it interferes with the regular bootstrap styling. I looked quickly through LORIS and don't see anywhere that is affected by this change. I also searched the codebase and H4 doesn't appear to be used anywhere, accept for in the CSS files.

### CODE REVIEW

**Comment by tedstrauss** [comment]
I've done a initial code review and didn't see any problems. I'd like to continue doing a thorough code review and feature test during the release cycle.

File: modules/training/ajax/getTrainingDoc.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+$finfo    = finfo_open(FILEINFO_MIME_TYPE);
+$MimeType = finfo_file($finfo, $FullPath);
+
+header("Content-type: $MimeType");
+$fp = fopen($FullPath, 'r');
+fpassthru($fp);
+fclose($fp);
+
+exit(5);
```
Comment (samirdas) [inline modules/training/ajax/getTrainingDoc.php]: Changed exit code to 0

**Comment by christinerogers** [comment]
This pull request introduced a permissions bug by changing the permissions IDs on timepoint_flag (14->) and timepoint_flag_evaluate (15->16).  The patch for existing projects didn't ensure that users who had previous permission IDs kept the permission when the permission IDs were changed.

**Comment by taracampbell** [comment]
Those changes are made in the default schema, not in the patch. The patch only adds a new permission. This wouldn't cause a bug would it?

File: SQL/0000-00-00-schema.sql [hunk 2]
```diff
@@ -982,8 +982,9 @@ INSERT INTO `permissions` VALUES
     (11,'data_entry','Data entry','1'),
     (12,'examiner_view','Add and certify examiners','2'),
     (13,'examiner_multisite','Across all sites add and certify examiners','2'),
-    (14,'timepoint_flag','Edit exclusion flags','2'),
-    (15,'timepoint_flag_evaluate','Evaluate overall exclusionary criteria for the timepoint','2'),
+    (14,'training','View and complete training','2'),
```
Comment (christinerogers) [inline SQL/0000-00-00-schema.sql]: New permissions are ideally added without shifting the permIDs of existing old permissions.  In theory this shouldn't matter but in practice it's easier if we don't make old projects' permIDs out of sync.

Comment (taracampbell) [inline SQL/0000-00-00-schema.sql]: But no old projects will be affected by this, as it is only in the default schema, so only for new installs.

**Pull request merged**